### DISCUSSION TITLE
How to disable indent lines?

### DISCUSSION THREAD

**Discussion by Houdiee** [zed-industries/zed#22285] [Feature Requests]
### Check for existing issues

- [X] Completed

### Describe the bug / provide steps to reproduce it

How can i disable the option to remove the gray lines for the tab indents on the left side of the code?

### Zed Version and System Specs

Zed: v0.161.2 (Zed)
OS: Linux Wayland nixos 24.11
Memory: 31.3 GiB
Architecture: x86_64
GPU: AMD Radeon RX 5700 XT (RADV NAVI10) || radv || Mesa 24.2.6

### If applicable, add screenshots or screencasts of the incorrect state / behavior

![image](https://github.com/user-attachments/assets/0e12b0d9-0143-4858-90e6-5c4e59fcd163)

### If applicable, attach your Zed.log file to this issue.

<details><summary>Zed.log</summary>

```

```
</details>

**Comment by SomeoneToIgnore** [comment]
https://github.com/zed-industries/zed/blob/1e2fa3b02209c9d51518c87c5324dfb0403c2b51/assets/settings/default.json#L290-L292

https://github.com/zed-industries/zed/blob/1e2fa3b02209c9d51518c87c5324dfb0403c2b51/assets/settings/default.json#L413-L422

https://github.com/zed-industries/zed/blob/1e2fa3b02209c9d51518c87c5324dfb0403c2b51/assets/settings/default.json#L447-L456

**Reply by Houdiee** [reply]
oops :sweat_smile:
